changelog/449.improvement.rst

```diff
@@ -0,0 +1 @@
+Use "yellow" main color with any XPASSED tests.
```

Comment:
IIUC there is no actual change here, because the previous code (probably unintentionally) used the fallback color, which is also yellow. So the changelog can be dropped as it might be confusing.

Reply:
..only if there is no passed one.
So now it is yellow always with an xpass.

Reply:
Oh right, I missed that this talks about the *main* color.